sdk/export/metric/aggregation/aggregation.go

```diff
@@ -58,23 +58,29 @@ type (
 		Max() (number.Number, error)
 	}
 
-	// Quantile returns an exact or estimated quantile over the
-	// set of values that were aggregated.
-	Quantile interface {
-		Aggregation
-		Quantile(float64) (number.Number, error)
-	}
-
 	// LastValue returns the latest value that was aggregated.
 	LastValue interface {
 		Aggregation
 		LastValue() (number.Number, time.Time, error)
 	}
 
-	// Points returns the raw set of values that were aggregated.
+	// Points returns the raw values that were aggregated.
 	Points interface {
 		Aggregation
-		Points() ([]number.Number, error)
+
+		// Points returns samples in the order they were
+		// recorded.  Points are approximately ordered by
+		// timestamp, but this is not guaranteed.
+		Points() (Samples, error)
+	}
+
+	// Samples is a list of Samples.
+	Samples []Sample
+
+	// Sample is a raw data point, consisting of a number and value.
+	Sample struct {
+		number.Number
+		time.Time
```

Comment:
It seems like we are mixing terminology here. Can we unify on `Point` or `Sample`?

Reply:
Yes, totally. I've replaced `Sample` with `Point` and `Samples` with `[]Point`. 

This remark made me realize something else leading to confusion here. The interface is named `Points` but my brain wants `Points` to be `[]Point`. 

This makes me want to rename all the interfaces in `sdk/export/metric/aggregation`, where `Points` is renamed `HasPoints`, `Sum` is renamed `HasSum`, and so on. Happy to do this in another PR if you agree, otherwise we have a bunch of interfaces with a method having the same name (my fault).

Reply:
Not all the interfaces though?  For example, `MinMaxSumCount` and `Distribution` are interfaces which do not have an eponymously named method.

I don't feel particularly strongly then about changing all the other aggregation interfaces to `Has<Aggregation>`